app/src/pages/inside/filtersPage/filterPageToolbar/filterPageToolbar.jsx

```diff
@@ -13,6 +13,8 @@ import styles from './filterPageToolbar.scss';
 
 const cx = classNames.bind(styles);
 
+const searchFilterValidator = (filter) => !filter || filter.length >= 3;
```

Comment:
May be isSearchFilterValid would provide more information about it's logic?

Reply:
@ElenaRomanchuk 
`isSearchFilterValid` sounds like a boolean variable, not a function.
What you think about `validateSearchFilter` ?

Reply:
I think it's OK